mobile/mobile-core/devices-service.ts

```diff
@@ -255,7 +255,7 @@ export class DevicesService extends EventEmitter implements Mobile.IDevicesServi
 	 * Method expects running devices.
 	 * @param identifier parameter passed by the user to --device flag
 	 */
-	private async getDevice(deviceOption: string): Promise<Mobile.IDevice> {
+	public async getDevice(deviceOption: string): Promise<Mobile.IDevice> {
```

Comment:
I agree that the method needs separate tests, but currently it is already covered with tests, mainly [here](https://github.com/telerik/mobile-cli-lib/blob/master/test/unit-tests/mobile/devices-service.ts#L505-L527)